docker/verify-c-common.Dockerfile

```diff
@@ -29,7 +29,7 @@ WORKDIR /home/usea/verify-c-common
 RUN rm -rf aws-c-common && git clone https://github.com/awslabs/aws-c-common.git
 
 WORKDIR /home/usea/verify-c-common/aws-c-common
-RUN mkdir build && cd build && cmake -DCMAKE_EXPORT_COMPILE_COMMANDS=1 -DCMAKE_INSTALL_PREFIX=$(pwd)/run ../ -GNinja && cmake --build . --target install
+RUN mkdir build && cd build && cmake -DCMAKE_C_COMPILER=clang-10 -DCMAKE_EXPORT_COMPILE_COMMANDS=1 -DCMAKE_INSTALL_PREFIX=$(pwd)/run ../ -GNinja && cmake --build . --target install
```

Comment:
don't you need to add extra flags for coverage, address sanitizer, and fuzzing instrumentation?

Reply:
good catch, would need those for coverage in CI to work

Reply:
added separate docker file